main/BUILD

```diff
@@ -4,12 +4,37 @@ load("@deps//:requirements.bzl", "requirement")
 py_binary(
     name = "train_classifier_g",
     srcs = ["train_classifier_g.py"],
+    visibility = ["//visibility:public"],
     deps = [
-        "//src:receive_data",
+        "//src:classifiers",
         "//src:download_data",
+        "//src:receive_data",
         "//src:suite",
-        "//src:classifiers",
-        requirement("pandas")
+        requirement("pandas"),
     ],
 )
 
+load(
+    "@com_grail_rules_r//R:defs.bzl",
+    "r_binary",
+    "r_library",
+    "r_pkg",
+    "r_pkg_test",
+    "r_unit_test",
+)
+load("@com_grail_rules_r//R:defs.bzl", "r_package")
+load("@com_grail_rules_r//R:defs.bzl", "r_package_with_test")
+load("@r_repositories_bzl//:r_repositories.bzl", "r_repositories")
+
+r_binary(
+    name = "feed_data",
+    src = "feed_data.R",
+    data = [
+        ":train_classifier_g",
```

Comment:
this doesn't go in deps, rather than data?

Reply:
If it goes to deps, it has to be something R.

Reply:
I suppose that makes sense. A data dependency seems strange, but oh well! :)